ee/clickhouse/system_status.py

```diff
@@ -43,6 +48,11 @@ def system_status() -> Generator[SystemStatusRow, None, None]:
         "metric": "Events recorded month to date",
         "value": get_event_count_month_to_date(),
     }
+    yield {
```

Comment:
Unsure if this is the best place to send the info, as we load this endpoint every time the app loads, could it be an expensive query? If so, I would send it in a separate endpoint we load when opening the licenses page.